### ISSUE TITLE
CRS 4.8.0 Rule ID 932236 remote cmd execusion FPs due to no space after command

### ISSUE THREAD

**Issue by stefan-kos** [coreruleset/coreruleset#3918]
Hi all,

in rule id 932236 there seems to be no space after the command which leads to false positives.
I see there are spaces in the regex, but they seem not to work. This is a huge one, I didn't get it where to start.

I use it with nginx, libmodsecurity 3.0.12, pcre2 10.44, CRS 4.8.0.

I have seen it because of a blocked request with c99 in a string of an argument which is a uuid, started with c99a5...

Thank you very much.

**Comment by Xhoenix** [comment]
Hi @stefan-kos ,

Thanks for reporting this. The false positive you reported is already known, and work is under progress to fix this. If you can provide the logs, e.g error and modsec_audit logs, then I'll be able to help you create a rule exclusion for this.

**Comment by stefan-kos** [comment]
Hi @Xhoenix , 

Thank you very much for your quick answer, very appreciated. I tried to search for and find if it's already known, but I could not find something, maybe my bad.

No worries, I made another own rule to cover cases where could affect my system and excluded the original one, just in case.

But 4.x ruleset works great, much less FPs even with standard PL3 without doing much. Love it.

Thank you very much for this good stuff, best regards, Stefan
